src/util/constants.ts

```diff
@@ -10,6 +10,11 @@ export const DEFAULT_LEADERBOARD_FORM = {
   season: 84,
 };
 
+export const DEFAULT_HISTORY_FORM = {
+  playerName: 'nhan385',
```

Comment:
Leave player name blank, new users I don't think should see this player name?

Reply:
done